server/users/userModel.js

```diff
@@ -1,27 +1,25 @@
-// needs to be updated.
-// used in server/users/controller.js
-// export const User = class User {
-//   constructor(username, email) {
-//     (this.username = username), (this.email = email);
-//   }
-//   toString() {
-//     return this.username + ', ' + this.email;
-//   }
-// }
+// User model
+// used in controller.js
+export const User = class User {
+  constructor(username, email) {
+    this.username = username;
+    this.email = email;
+  }
 
-// // https://firebase.google.com/docs/firestore/manage-data/add-data
-// // need to convert class object to be compatible with Firestore
-// export const userConverter = {
-//   toFirestore: (user) => {
-//     return {
-//       username: user.username,
-//       email: user.email,
-//     };
-//   },
-//   fromFirestore: (snapshot, options) => {
-//     const data = snapshot.data(options);
-//     return new User(data.username, data.email);
-//   },
-// };
+  toString() {
+    return `${this.username}, ${this.email}`;
+  }
+};
```

Comment:
Class-based `User` seems really unnecessary for what we need - what are your thoughts on adopting TypeScript instead so we have the `User` interface enforced better 

I can't see us requiring many OOP methods on the `User` class